addons/web/static/src/views/list/list_renderer.js

```diff
@@ -675,6 +675,9 @@ export class ListRenderer extends Component {
             if (record.isReadonly(column.name)) {
                 classNames.push("o_readonly_modifier");
             }
+            if (record.isInvisible(column.name)) {
+                classNames.push("o_invisible_modifier");
+            }
```

Comment:
```suggestion
            if (record.isReadonly(column.name) && !record.isInvisible(column.name)) {
                classNames.push("o_readonly_modifier");
            }
```

Reply:
I thought we need to remove underline when field is required and invisible(actually values are hidden) that why I did this [issue](https://drive.google.com/file/d/1KzSS8j8sR-aqE-3x2kNihnH-oTWxJPQq/view?usp=sharing) but it'll not solve issue in video cause it was different before 16.0.

And attrs `readonly`  in xml was not to hide underline I added it before list converted in OWL. I checked it `invisible` attrs is good enough to remove underline. so I'll remove `readony` attrs for arch

may be first one is real issue could you check?

Thank you :slightly_smiling_face:

Reply:
I am not sure I got it, I'll have a look with @xavierbol tomorrow.
Thanks